api/HashChangeEvent.json

```diff
@@ -0,0 +1,154 @@
+{
+  "api": {
+    "HashChangeEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HashChangeEvent",
+        "support": {
+          "webview_android": {
+            "version_added": "2.2"
```

Comment:
Change this to `true`. Webview follows Chrome version numbers, not Android version numbers. (It's probably 5, but I can't confirm something that old, so `true`.)